lib/active_admin/application.rb

```diff
@@ -216,7 +216,8 @@ def routes(rails_router)
     # Example usage:
     #   ActiveAdmin.before_filter :authenticate_admin!
     #
-    %w(before_filter skip_before_filter after_filter skip_after_filter around_filter skip_filter).each do |name|
+    AbstractController::Callbacks::ClassMethods.public_instance_methods.
+        find_all { |m| m.match(/(filter|action)/) }.each do |name|
```

Comment:
Prefer `select` over `find_all`.<br>Use 2 (not 4) spaces for indenting an expression spanning multiple lines.